convene-web/app/helpers/rooms_helper.rb

```diff
@@ -0,0 +1,17 @@
+module RoomsHelper
+  def end_call_path(workspace)
+    if workspace.branded_domain.present?
+      "https://#{workspace.branded_domain}/"
+    else
+      workspace_path(workspace)
+    end
+  end
```

Comment:
These custom paths methods indicate to me that we’re “going off the Rails.” I’d rather we rely on `workspace_path` and don’t create custom path methods unless necessary

Reply:
I feel the same way but also saw `doorkeeper` was doing something similar so I thought it's ok.
https://github.com/doorkeeper-gem/doorkeeper/blob/2fd1556ade7572d01e507c0fbc53eaf6c5bbfd19/app/helpers/doorkeeper/dashboard_helper.rb#L17-L19

Reply:
One thing I don't like about this `branded_domain` route is it's making smoke test locally more difficult, I am more concern on how we write e2e test on this.

Reply:
Yea, that's why I'm trying to make sure it's something that _only_ exists as an entrance point into the system; and _not_ as something that the system tries to maintain consistency on. Essentially it's something you put on the business card; but it's not something we project back into the world.